_pages/webextensions.md

```diff
@@ -12,160 +12,173 @@ parent: activities
 <div class="col-md-3 hidden-xs pull-right" markdown="1">
 ## Impact and Metrics
 
-Add-ons provide a safe, simple, and powerful way to personalize your browsing experience. With WebExtensions APIs, it’s easier than ever to create extensions that can be easily ported to Firefox, Chrome, Edge, and Opera. Throwing a workshop helps more people learn how to develop add-ons and help ensure that Firefox remains the most customizable browser.
+Throwing a workshop helps more people learn how to develop add-ons and help ensure that Firefox remains the most customizable browser.
+
 
 ### Campaign Goals
 
 __Impact:__
 
 * 125 developers are introduced to and building browser extensions either online or at regional events.
-* 50 add-ons tagged with mozactivate17 in addons.mozilla.org.
-* 50 social media posts (Twitter, Facebook, blogs, etc.) that reference extension development and #MozActivate.
+* 20 add-ons tagged with ‘#mozactivate17’ in addons.mozilla.org.
 
 __Strength:__
 
-* 25 Mozillians organize Build Your Own Extension workshops
+* 15 Mozillians organize Build Your Own Extension workshops 
```

Comment:
I assume those will be measured against activities happening in 2018?

Reply:
Yes, they will. :)